### PR TITLE
first draft of FEDERATION.md

### PR DESCRIPTION
**Pull request by pfefferle**
See FEP-67ff: FEDERATION.md https://codeberg.org/fediverse/fep/src/branch/main/fep/67ff/fep-67ff.md

### CODE REVIEW

File: FEDERATION.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- [ActivityPub](https://www.w3.org/TR/activitypub/) (Server-to-Server)
+- [WebFinger](https://webfinger.net/)
+- [Http Signatures](https://datatracker.ietf.org/doc/html/draft-cavage-http-signatures)
+- [NodeInfo](https://nodeinfo.diaspora.software/)
+
+## Supported FEPs
+
+- [FEP-f1d5: NodeInfo in Fediverse Software](https://codeberg.org/fediverse/fep/src/branch/main/fep/f1d5/fep-f1d5.md)
+- [FEP-67ff: FEDERATION.md](https://codeberg.org/fediverse/fep/src/branch/main/fep/67ff/fep-67ff.md)
+- [FEP-5feb: Search indexing consent for actors](https://codeberg.org/fediverse/fep/src/branch/main/fep/5feb/fep-5feb.md)
```
Comment (jeherve) [inline FEDERATION.md:16]: How is this handled in the plugin? Isn't every author set to ` "indexable": true` by default?

File: FEDERATION.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Federation in WordPress
+
+The WordPress plugin largely follows ActivityPub's server-to-server specification, but makes use of some non-standard extensions, some of which are required to interact with the plugin. Most of these extensions are for the purpose of compatibility with other, sometimes very restrictive networks, such as Mastodon.
+
+## Supported federation protocols and standards
+
+- [ActivityPub](https://www.w3.org/TR/activitypub/) (Server-to-Server)
+- [WebFinger](https://webfinger.net/)
+- [Http Signatures](https://datatracker.ietf.org/doc/html/draft-cavage-http-signatures)
```
Comment (jeherve) [inline FEDERATION.md:9]:
```suggestion
- [HTTP Signatures](https://datatracker.ietf.org/doc/html/draft-cavage-http-signatures)
```

File: FEDERATION.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+More information on HTTP Signatures, as well as examples, can be found here: https://docs.joinmastodon.org/spec/security/#http
+
+## Additional documentation
+
+- Plugin Description: https://github.com/Automattic/wordpress-activitypub?tab=readme-ov-file#description
+- Frequently Asked Questions: https://github.com/Automattic/wordpress-activitypub?tab=readme-ov-file#frequently-asked-questions
+- Installation Instructions: https://github.com/Automattic/wordpress-activitypub?tab=readme-ov-file#installation
+- Upgrade Notice: https://github.com/Automattic/wordpress-activitypub?tab=readme-ov-file#upgrade-notice
+- Changelog: https://github.com/Automattic/wordpress-activitypub?tab=readme-ov-file#changelog
```
Comment (jeherve) [inline FEDERATION.md:38]: This reminds me of #439. I wonder if some of this (namely the changelog) wouldn't be better off in its own file at this point?

File: FEDERATION.md (hunk 1 above)
Comment (pfefferle) [reply FEDERATION.md:16]: It's depending on the "Search engine visibility" setting: https://github.com/Automattic/wordpress-activitypub/blob/master/includes/model/class-user.php#L312

File: FEDERATION.md (hunk 3 above)
Comment (pfefferle) [reply FEDERATION.md:38]: If these were generated automatically based on the readme.txt or vice versa, I am totally fine :)

**Review by jeherve — APPROVED** [review]

**Pull request merged**